deploy/deploy.sh

```diff
@@ -168,7 +171,14 @@ rm -r $DEST_FOLDER "$DEST_FOLDER.zip";
 # Finish
 #
 cat << FINMSG
-+----------------------+
-| Finished installing. |
-+----------------------+
+   ______      ____                   ______      __
```

Comment:
The backticks ` need to be changed here, as in bash, they're treated as "execute what's inside them"